tests/verify_override_true_profile.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+import argparse
+import lxml.etree as ET
```

Comment:
At this moment lxml is optional and we still prefer xmltree instead. I think there is nothing in this test that would require lxml and you can achieve the same thing by using xmltree.